src/docs/content/docs/configuration/reaper_specific.md

```diff
@@ -0,0 +1,343 @@
+
+The output format of an entry in the log.
+
+</br>
+
+### `metrics`
+
+Metrics configuration parameters for sending metrics to a reporting system via the [Dropwizard interface](http://www.dropwizard.io/1.1.4/docs/manual/configuration.html#metrics) .
+
+    metrics:
+      frequency: "1 minute"
+      reporters:
+        - type: <type>
+
+</br>
+
+### `repairIntensity`
+
+Type: *Float* (value between 0.0 and 1.0, but must never be 0.0.)
+
+Repair intensity defines the amount of time to sleep between triggering each repair segment while running a repair run. When intensity is 1.0, it means that Reaper doesn't sleep at all before triggering next segment, and otherwise the sleep time is defined by how much time it took to repair the last segment divided by the intensity value. 0.5 means half of the time is spent sleeping, and half running. Intensity 0.75 means that 25% of the total time is used sleeping and 75% running. This value can also be overwritten per repair run when invoking repairs.
+
+</br>
+
+### `repairManagerSchedulingIntervalSeconds`
+
+Type: *Integer*
+
+Default: *30*
+
+Controls the pace at which the Repair Manager will schedule processing of the next segment. Reducing this value from its default value of 30s to a lower value can speed up fast repairs by orders of magnitude.
+
+</br>
+
+### `repairParallelism`
+
+Type: *String*
+
+Type of parallelism to apply by default to repair runs. The value must be either **SEQUENTIAL**, **PARALLEL**, or **DATACENTER_AWARE**.
+
+**SEQUENTIAL** - one replica at a time, validation compaction performed on snapshots
+
+**PARALLEL** - all replicas at the same time, no snapshot
+
+**DATACENTER_AWARE** - all replicas in only one DC at a time, no snapshots. If this value is used in clusters older than 2.0.12, Reaper will fall back into using **SEQUENTIAL** for those clusters.
+
+</br>
+
+### `repairRunThreadCount`
+
+Type: *Integer*
+
+The amount of threads to use for handling the Reaper tasks. Have this big enough not to cause
+blocking in cause some thread is waiting for I/O, like calling a Cassandra cluster through JMX.
+
+</br>
+
+### `scheduleDaysBetween`
+
+Type: *Integer*
+
+Default: *7*
+
+Defines the amount of days to wait between scheduling new repairs. The value configured here is the default for new repair schedules, but you can also define it separately for each new schedule. Using value 0 for continuous repairs is also supported.
+
+</br>
+
+### `segmentCount`
+
+Type: *Integer*
+
+Defines the default amount of repair segments to create for newly registered Cassandra repair runs (token rings). When running a repair run by the Reaper, each segment is repaired separately by the Reaper process, until all the segments in a token ring are repaired. The count might be slightly off the defined value, as clusters residing in multiple data centers require additional small token ranges in addition to the expected. This value can be overwritten when executing a repair run via Reaper.
+
```

Comment:
There's a hidden `segmentCountPerNode` setting now that replaces the legacy `segmentCount`.

Reply:
Ok. I will document that instead.

Reply:
Ok. I'll mark this as deprecated. We can discuss if we want to document the hidden `segmentCountPerNode` at a later point.